src/main/java/uk/gov/hmcts/reform/blobrouter/reconciliation/model/in/SupplierStatement.java

```diff
@@ -12,7 +12,7 @@
     @NotNull
     public final List<Envelope> envelopes;
 
-    public SupplierStatement(@JsonProperty("envelopes") List<Envelope> envelopes) {
+    public SupplierStatement(@JsonProperty(value = "envelopes", required = true) List<Envelope> envelopes) {
```

Comment:
Can this be set in an annotation on the field instead?
There is one already (`@NotNull`)

Reply:
This is no longer required. it validates with `@NotNull` annotation on the field